packages/protocol/package.json

```diff
@@ -6,61 +6,58 @@
-    "gas": "yarn run test --gas",
-    "pull-submodules": "git submodule update --init --recursive",
-    "delete-submodules": "rm -rf $(git submodule | awk '{ print $2 }')",
-    "build:sol": "yarn pull-submodules && mkdir -p migrations && yarn ts-node --preferTsExts ./scripts/build.ts --solidity ${BUILD_DIR:-./build}",
-    "build": "yarn build:sol && yarn build:ts",
-    "prebuild": "rm -rf ./build",
-    "determine-release-version": "yarn --silent ts-node --preferTsExts ./scripts/determine-release-version.ts",
-    "prepare_contracts_and_abis_publishing": "yarn ts-node --preferTsExts ./scripts/prepare-contracts-and-abis-publishing.ts",
-    "prepare_devchain_anvil_publishing": "yarn ts-node --preferTsExts ./scripts/change-anvil-devchain-package-version.ts",
-    "validate_abis_exports": "yarn ts-node --preferTsExts ./scripts/validate-abis-package.ts",
-    "sourcify-publish": "yarn ts-node ./scripts/sourcify-publish.ts",
-    "migrate": "./scripts/bash/migrate.sh",
-    "set_block_gas_limit": "./scripts/bash/set_block_gas_limit.sh",
-    "download-artifacts": "./scripts/bash/download_artifacts.sh",
-    "init-network": "./scripts/bash/init_network.sh",
-    "upload-artifacts": "./scripts/bash/upload_artifacts.sh",
-    "upgrade": "./scripts/bash/upgrade.sh",
-    "revoke": "./scripts/bash/revoke.sh",
-    "govern": "./scripts/bash/govern.sh",
-    "console": "./scripts/bash/console.sh",
-    "check-versions": "./scripts/bash/check-versions.sh",
-    "check-opcodes": "yarn ts-node scripts/check-opcodes.ts",
-    "make-release": "./scripts/bash/make-release.sh",
-    "verify-deployed": "./scripts/bash/verify-deployed.sh",
-    "verify-release": "./scripts/bash/verify-release.sh",
-    "size:onchain": "./scripts/bash/get_smart_contract_size_from_onchain_address.sh",
-    "size:artifacts": "./scripts/bash/get_smart_contract_size_from_build_artifacts.sh",
+    "devchain": "yarn ts-node scripts/devchain.ts",
+    "devchain:init-network": "./scripts/bash/init_network.sh",
+    "devchain:generate-old-devchain-and-build": "./scripts/bash/generate-old-devchain-and-build.sh",
+    "devchain:migrate": "./scripts/bash/migrate.sh",
+    "devchain:reset": "yarn devchain generate-tar .tmp/devchain.tar.gz --upto 29",
     "ganache-dev": "./scripts/bash/ganache.sh",
     "ganache-devchain": "./scripts/bash/ganache_devchain.sh",
+    "lint": "yarn run lint:ts && yarn run lint:sol",
+    "lint:sol": "solhint --version && solhint './contracts/**/*.sol' && solhint './contracts-0.8/**/*.sol'",
+    "lint:ts": "yarn run --top-level eslint .",
+    "release:check-opcodes": "yarn ts-node scripts/check-opcodes.ts",
+    "release:check-versions": "./scripts/bash/check-versions.sh",
+    "release:determine-release-version": "yarn --silent ts-node --preferTsExts ./scripts/determine-release-version.ts",
+    "release:make-release": "./scripts/bash/make-release.sh",
```

Comment:
if these commands are changing, we also need to update the docs [here](https://github.com/celo-org/docs/blob/main/docs/community/release-process/smart-contracts.md)

Additionally, I think it'd be worth searching all the renamed commands in the docs repo to update them there as well

Reply:
There is companion PR in docs repo: https://github.com/celo-org/docs/pull/1878

Reply:
They are up to date now 😎 
Resolving 👍